src/components/VideoCart.tsx

```diff
@@ -44,7 +44,7 @@ const VideoCart = (props: videoCartProps) => {
     return (
         <div className={`${singleColumn ? ' sm:w-full p-2' : 'sm:w-1/2 xl:w-1/4 p-2'}`}>
             <div>

```

Comment:
why we have 2 divs with no classes or anything we need to use the least amount of elements?